### ISSUE TITLE
[Component] [pagination] el-pagination警告

### ISSUE THREAD

**Issue by itlzbin** [element-plus/element-plus#16736]
Bug Type: **`Component`**

## Environment

- Vue Version: `3.4.26`
- Element Plus Version: `2.7.2`
- Browser / OS: `Google Chrome 版本 119.0.6045.159（正式版本） (arm64)`
- Build Tool: `Vite`

### Related Component

- `el-pagination`

### Reproduction Link

[Element Plus Playground](https://element-plus.run/#eyJzcmMvQXBwLnZ1ZSI6IjxzY3JpcHQgc2V0dXAgbGFuZz1cInRzXCI+XG5pbXBvcnQgeyByZWYsIHZlcnNpb24gYXMgdnVlVmVyc2lvbiB9IGZyb20gJ3Z1ZSdcbmltcG9ydCB7IHZlcnNpb24gYXMgZXBWZXJzaW9uIH0gZnJvbSAnZWxlbWVudC1wbHVzJ1xuaW1wb3J0IHsgRWxlbWVudFBsdXMgfSBmcm9tICdAZWxlbWVudC1wbHVzL2ljb25zLXZ1ZSdcblxuY29uc3QgY3VycmVudFBhZ2UxID0gcmVmKDUpXG5jb25zdCBjdXJyZW50UGFnZTIgPSByZWYoNSlcbmNvbnN0IGN1cnJlbnRQYWdlMyA9IHJlZig1KVxuY29uc3QgY3VycmVudFBhZ2U0ID0gcmVmKDQpXG5jb25zdCBwYWdlU2l6ZTIgPSByZWYoMTAwKVxuY29uc3QgcGFnZVNpemUzID0gcmVmKDEwMClcbmNvbnN0IHBhZ2VTaXplNCA9IHJlZigxMDApXG5jb25zdCBzbWFsbCA9IHJlZihmYWxzZSlcbmNvbnN0IGJhY2tncm91bmQgPSByZWYoZmFsc2UpXG5jb25zdCBkaXNhYmxlZCA9IHJlZihmYWxzZSlcblxuY29uc3QgaGFuZGxlU2l6ZUNoYW5nZSA9ICh2YWw6IG51bWJlcikgPT4ge1xuICBjb25zb2xlLmxvZyhgJHt2YWx9IGl0ZW1zIHBlciBwYWdlYClcbn1cbmNvbnN0IGhhbmRsZUN1cnJlbnRDaGFuZ2UgPSAodmFsOiBudW1iZXIpID0+IHtcbiAgY29uc29sZS5sb2coYGN1cnJlbnQgcGFnZTogJHt2YWx9YClcbn1cbjwvc2NyaXB0PlxuXG48dGVtcGxhdGU+XG4gIDxkaXYgY2xhc3M9XCJkZW1vLXBhZ2luYXRpb24tYmxvY2tcIj5cbiAgICA8ZGl2IGNsYXNzPVwiZGVtb25zdHJhdGlvblwiPkFsbCBjb21iaW5lZDwvZGl2PlxuICAgIDxlbC1wYWdpbmF0aW9uXG4gICAgICB2LW1vZGVsOmN1cnJlbnQtcGFnZT1cImN1cnJlbnRQYWdlNFwiXG4gICAgICB2LW1vZGVsOnBhZ2Utc2l6ZT1cInBhZ2VTaXplNFwiXG4gICAgICA6cGFnZS1zaXplcz1cIlsxMDAsIDIwMCwgMzAwLCA0MDBdXCJcbiAgICAgIDpzbWFsbD1cInNtYWxsXCJcbiAgICAgIDpkaXNhYmxlZD1cImRpc2FibGVkXCJcbiAgICAgIDpiYWNrZ3JvdW5kPVwiYmFja2dyb3VuZFwiXG4gICAgICBsYXlvdXQ9XCJ0b3RhbCwgc2l6ZXMsIHByZXYsIHBhZ2VyLCBuZXh0LCBqdW1wZXJcIlxuICAgICAgOnRvdGFsPVwiNDAwXCJcbiAgICAgIEBzaXplLWNoYW5nZT1cImhhbmRsZVNpemVDaGFuZ2VcIlxuICAgICAgQGN1cnJlbnQtY2hhbmdlPVwiaGFuZGxlQ3VycmVudENoYW5nZVwiXG4gICAgLz5cbiAgPC9kaXY+XG48L3RlbXBsYXRlPlxuIiwiaW1wb3J0LW1hcC5qc29uIjoie1xuICBcImltcG9ydHNcIjoge31cbn0iLCJ0c2NvbmZpZy5qc29uIjoie1xuICBcImNvbXBpbGVyT3B0aW9uc1wiOiB7XG4gICAgXCJ0YXJnZXRcIjogXCJFU05leHRcIixcbiAgICBcImpzeFwiOiBcInByZXNlcnZlXCIsXG4gICAgXCJtb2R1bGVcIjogXCJFU05leHRcIixcbiAgICBcIm1vZHVsZVJlc29sdXRpb25cIjogXCJCdW5kbGVyXCIsXG4gICAgXCJ0eXBlc1wiOiBbXCJlbGVtZW50LXBsdXMvZ2xvYmFsLmQudHNcIl0sXG4gICAgXCJhbGxvd0ltcG9ydGluZ1RzRXh0ZW5zaW9uc1wiOiB0cnVlLFxuICAgIFwiYWxsb3dKc1wiOiB0cnVlLFxuICAgIFwiY2hlY2tKc1wiOiB0cnVlXG4gIH0sXG4gIFwidnVlQ29tcGlsZXJPcHRpb25zXCI6IHtcbiAgICBcInRhcmdldFwiOiAzLjNcbiAgfVxufVxuIiwiX28iOnt9fQ==)

### Steps to reproduce

```vue
<el-pagination
      v-model:current-page="currentPage4"
      v-model:page-size="pageSize4"
      :page-sizes="[100, 200, 300, 400]"
      :small="small"
      :disabled="disabled"
      :background="background"
      layout="total, sizes, prev, pager, next, jumper"
      :total="400"
      @size-change="handleSizeChange"
      @current-change="handleCurrentChange"
    />
```

### What is Expected?

无警告

### What is actually happening?

```vue
error.ts:14 ElementPlusError: [el-input] [API] label is about to be deprecated in version 2.8.0, please use aria-label instead.
For more detail, please visit: https://element-plus.org/en-US/component/input.html

    at debugWarn (error.ts:13:37)
    at watch.immediate (index.ts:6:7)
    at callWithErrorHandling (runtime-core.esm-bundler.js:195:19)
    at callWithAsyncErrorHandling (runtime-core.esm-bundler.js:202:17)
    at job (runtime-core.esm-bundler.js:1955:9)
    at doWatch (runtime-core.esm-bundler.js:1993:7)
    at watch (runtime-core.esm-bundler.js:1820:10)
    at useDeprecated (index.ts:4:3)
    at setup (input.vue:551:3)
    at callWithErrorHandling (runtime-core.esm-bundler.js:195:19)
```

### Additional comments

"layout" attribute value, removing "jumper" is normal, but using "jumper" will cause a warning.
-------------------------------------------------
属性：layout="total, sizes, prev, pager, next, jumper"  
如果不加jumper则没有警告，如果加上了jumper则会出现警告

<img width="806" alt="image" src="https://github.com/element-plus/element-plus/assets/168640044/c578d385-453c-48bd-9fbb-ab66d4aace51">

**Comment by pinuo520** [comment]
me too

**Comment by btea** [comment]
fixed by #16682

**Comment by Ale6333499** [comment]
2.7.2还有这个问题
